ghost/core/core/server/services/link-tracking/PostLinkRepository.js

```diff
@@ -48,6 +49,29 @@ module.exports = class PostLinkRepository {
         return result;
     }
 
+    async updateLinks(linkIds, updateData, options) {
+        const bulkUpdateOptions = _.pick(options, ['transacting']);
+
+        const bulkActionResult = await this.#LinkRedirect.bulkEdit(linkIds, 'redirects', {
```

Comment:
Our bulkX methods will allow some operations to fail, are we okay with a partial edit from a product POV?

Reply:
Think we are ok with that atm, as ideally unlike bulk members the links don't have any dependency/checks before the bulk operation so they shouldn't fail, and if they do we can flag up on UI I guess as we have access to all links, but I'll double check on this